init/init_sec.cpp

```diff
@@ -48,6 +48,8 @@ static device_variant parse_variant(std::string bl) {
         ret = VARIANT_G955L;
     else if (bl.find("G955W8") != std::string::npos)
         ret = VARIANT_G955W8;
+    else if (bl.find("N950F") != std::string::npos)
+        ret = VARIANT_G955W8;
```

Comment:
needs to be ```VARIANT_N950F``` not ```VARIANT_G955W8```

Reply:
Ah yeah, sorry ;) lemme fix it and repush :D